admin/python/lsst/qserv/testing/README.md

```diff
@@ -0,0 +1,164 @@
+
+# Qserv KPM test harness
+
+This package contains Python code and configuration examples for PM testing of
+QServ cluster. Main application is `qserv-kraken` (lives in `admin/bin/`) and
```

Comment:
I'm probably lame but I don't know what PM is. Avoid acronyms in readme?

Reply:
and again, probably I'm dense but maybe it's worth saying "Main application is a command line application called `qserv-kraken`..." I only realized on line 84 that it's a CLI app 🙃

Reply:
PM is KPM with missing K. Sorry for that, but my spell checker was OK with PM (it probably assumed I'm talking about Prime Minister or something) 😄 I guess we all should know what KPM is? 

I'll expand on "main application".